src/app.ts

```diff
@@ -23,7 +23,10 @@ import {
 import { pad, padStart, sample, padEnd } from 'lodash';
 import chalk from 'chalk';
 
-app();
+app().catch(e => {
+  logger.error(e.message);
```

Comment:
Do you think it might make sense here to just say something about how SUSHI exited unexpectedly along with the `e.message`? In theory we shouldn't ever get here anyway. But I'm thinking that if an error actually does sneak its way up here, it's possible the message associated with it is confusing/not helpful.

Reply:
Sure! That seems reasonable. How do you feel about
> SUSHI encountered the following unexpected error: ${error.message}

Reply:
Looks good to me!